server/app/lib/fusor/validators/deployment_validator.rb

```diff
@@ -79,6 +80,19 @@ def validate(deployment)
           if !deployment.rhev_is_self_hosted && deployment.rhev_hypervisor_hosts.count < 1
             deployment.errors[:rhev_hypervisor_hosts] << _('RHEV deployments must have at least one Hypervisor')
           end
+
+          special = "?<>',?[]}{=-)(*&^%$#`~{}"
+          regex = /[#{special.gsub(/./) { |char| "\\#{char}" }}]/
+
+          if !(deployment.rhev_engine_host.name =~ regex).nil?
```

Comment:
`nil` is falsey in ruby so you don't need to check for it. This works just fine:

```ruby
deployment.rhev_engine_host.name =~ regex
```